### ISSUE TITLE
Not possible to log in in FoxApp when being in a different network than foxbox

### ISSUE THREAD

**Issue by isabelrios** [fxbox/app#64]
Foxbox is running in one network, launch https://fxbox.github.io/app from a device connected to a diffferent network. The log in page is loaded but when tapping on Log in button the Set up page is not shown instead it appears an error to connect to the site.

cc @JohanLorenzo, @npark-mozilla

**Comment by michielbdejong** [comment]
This is by design - the first time you connect *has* to be on the same network. If this wasn't clear to you in the UI we should make the 'No Boxes found on current network' message clearer (so far we've mainly developed the 'happy path', some work is still to be done on error messages like that one.

Once the app has connected successfully with a given Box at least once, it can ([for certain tasks, not necessarily all tasks](https://github.com/fxbox/RFC/pull/2)) connect to the Box from any other network, via the pagekite tunnel (if activacted).

I'll close this bug as wontfix - feel free to reopen if you think this was in error! :)

**Comment by isabelrios** [comment]
Thank you Michiel,
So far I do not see any message like that in the UI when trying to open the app for the first time in a different network.
Also, althought I do the set up in the same network, the next time I want to access the app from a different network, this problem also exists. I have tunneling enabled, but still the app seems not to be accesible.
I agree on closing this issue and maybe opening two different ones:
1- To follow that there is no error message first time accessing the app from a different network (or add more information on [issue 9](https://github.com/fxbox/app/issues/9)
2- To follow that the app is not accessible (once the set up is done) from a different network
What do you think? 
Thanks!
